ui/src/Components/Dashboard/TableRow/TableRow.js

```diff
@@ -0,0 +1,19 @@
+import Toggle from "../../Global/Toggle/Toggle"
+
+export default function TableRow({ featureFlag, headers }) {
+  return (
+    <tr
+      className='border-t'
+    >
+      {headers.map((heading, index) =>
+        <td className='px-4 py-2' key={index}>{featureFlag[heading]}</td>
```

Comment:
I am assuming this is WIP as of now. He need to add the tags for the text and that will depend on the type of content we will have in the table

Reply:
Sorry. I'm not getting what you mean.

Reply:
So, what we are planning is that we pass tags as props to enable that particular column?